src/backend/distributed/shared_library_init.c

```diff
@@ -287,6 +290,52 @@ static const struct config_enum_entry create_object_propagation_options[] = {
 	{NULL,        0,                                   false}
 };
 
+static const struct config_enum_entry cpu_priority_options[] = {
+	{ "unchanged", CPU_PRIORITY_UNCHANGED, false },
```

Comment:
wonder if there is a more typical name, e.g. `inherit` or `default` , not sure

Reply:
changed to inherit